conf.d/mapreduce.yaml.example

```diff
@@ -0,0 +1,23 @@
+instances:
+    -   resourcemanager_uri: http://localhost:8088
+
+init_config:
+  #
+  # Return metrics for specified counters
+  #
+
+  counter_metrics:
+  #   - job_name: 'Foo'
```

Comment:
Could you add a comment here in the yaml file with a link to the related hadoop doc page? (probably https://hadoop.apache.org/docs/current/hadoop-mapreduce-client/hadoop-mapreduce-client-core/MapredAppMasterRest.html#Job_Counters_API)